Sources/ComposableArchitecture/Documentation.docc/ComposableArchitecture.md

```diff
@@ -72,7 +72,6 @@ day-to-day when building applications, such as:
 
 ### Testing
 
-- <doc:Testing>
```

Comment:
We link to this in the same section under the "Essentials", so this felt a little redundant...